rdflib/__init__.py

```diff
@@ -84,6 +97,9 @@
 ]
 
 import sys
+from rdflib.namespace import XMLNS, SOSA
+
+assert sys.version_info >= (2, 7, 0), "rdflib requires Python 2.7 or higher"
```

Comment:
I don't think this check is needed since 6.0.0 will drop 2.7 support and because any unsupported versions should be caught before this assertion is ever reached.